components/mediation/security/org.wso2.micro.integrator.mediation.security/src/main/java/org/wso2/micro/integrator/mediation/security/vault/SecureVaultConstants.java

```diff
@@ -42,4 +44,20 @@
 	String CONF_CONNECTOR_SECURE_VAULT_CONFIG_PROP_LOOK =
 	                                                                          "conf:/repository/components/secure-vault";
 
+	/**
+	 * System property to specify customized docker secret root directory
+	 */
+	public static final String PROP_DOCKER_SECRET_ROOT_DIRECTORY = "ei.secret.docker.root.dir";
```

Comment:
Do we need to make this as mi or keep as ei itself ?

Reply:
I think we don't need to have the word ei/mi, WDYT?